### ISSUE TITLE
Make headings in sidebar & footer more visually distinct

### ISSUE THREAD

**Issue by tobyhodges** [carpentries/carpentries.org#313]
Feedback from Maintainers at a recent subcommunity meeting.

The headings in the page footers (see screengrab below) and the right sidebar (e.g. Table of Contents) are not as easy to distinguish from the links underneath them. This can make it difficult to predict what will be clickable and what will not.

![page-footer-heading-highlighted](https://github.com/user-attachments/assets/efe18186-7cab-4b62-a026-69fc2ff114d2)

![sidebar-toc-heading](https://github.com/user-attachments/assets/05504838-d2f9-4f40-a9fb-860c3ad0c11e)

**Comment by maneesha** [comment]
Thanks for this feedback @tobyhodges 
For the Table of Contents - that should be hidden on any page that doesn't have sections that would merit a ToC (like the screenshot you shared above).  Can you share where you still see it? 

For the footer menus - tagging @OscarSiba  for suggestions on styling that may improve this UI.

**Comment by tobyhodges** [comment]
Thanks @maneesha. The ToC heading was found on https://carpentries.org/search/

**Comment by maneesha** [comment]
The ToC is now hidden on the [search page](https://carpentries.org/search/).

**Comment by OscarSiba** [comment]
Hi @maneesha. Is it possible to make the footer links the same colour blue as in this button: 
![image](https://github.com/user-attachments/assets/24da62b2-d8fd-49b0-879a-51b438a68343)

This will also apply to the lesson programs' websites footers.

**Comment by maneesha** [comment]
This is what that would look like:
![recolor-footer-menu](https://github.com/user-attachments/assets/f90bd540-c5a7-4057-b8ab-729c5e1ffaa4)
@OscarSiba let me know what you think.  Note that this would require a change to the theme, and I would not be able to get to that right away.

**Comment by maneesha** [comment]
[Consider accessibility](https://coolors.co/contrast-checker/3b82f6-383838)

**Comment by OscarSiba** [comment]
Hmmm... I think perhaps let's park this for now @maneesha and revisit in January/Q1 when looking at all teh feebdack we have received from the community and when we focus on the site-wide look and feel additions, taking into account all the accessibility concerns.

**Comment by maneesha** [comment]
@OscarSiba suggests making these fire red to match the donate button. (Check for accessibility first)

**Comment by maneesha** [comment]
@OscarSiba This is what it would look like with fire red (#FF4955)

![Image](https://github.com/user-attachments/assets/4aecd42b-be52-44f9-aea3-1518b224937b)

See also the [accessibility contrast check](https://coolors.co/contrast-checker/ff4955-383838).

**Comment by OscarSiba** [comment]
@maneesha can I confirm if the colour for the current footer links text is #FFF7F1 or not? 

If it is, that is one of our brand colors and it should work well for accessibility. In which case, are we able to underline the links so it's a bit clearer that they're hyperlinks that are clickable?

**Comment by maneesha** [comment]
Yes, the current color is #FFF7F1.
Here is what it would look like if we kept the color and underlined the links:

![Image](https://github.com/user-attachments/assets/77fb13a6-e274-42a3-a585-02743f522af5)

**Comment by OscarSiba** [comment]
Then let's underline the links, please, and keep the colour the same.

**Comment by maneesha** [comment]
@OscarSiba Here's another one where I did not ask enough questions.
Currently, the links get underlined when you mouse over them. If we had them underlined by default, then there would be no change when you mouse over them. Let me know if that is what you want or if you are thinking of anything else.

**Comment by OscarSiba** [comment]
Thanks @maneesha. Is it possible to underline them, but w=then when you hover on them they turn blue (similar to what happens with the blog posts titles)? Or is this a different design element all together?

**Comment by maneesha** [comment]
Yes, although please see the note above about accessibility when using blue.

**Comment by OscarSiba** [comment]
Hi Maneesha. Then with this one, please permanently underline them as they currently are, and when you hover over the links please turn them our yellow (#FFC700). Thank you!

**Comment by maneesha** [comment]
@OscarSiba  Here is a preview of what that would look like.

https://github.com/user-attachments/assets/6b24f13d-1cca-4b36-a351-9cf1271b9b03

**Comment by OscarSiba** [comment]
That's perfect, thanks @maneesha, please merge.

**Comment by maneesha** [comment]
Addressed with change to theme.
